script/main.go

```diff
@@ -1,7 +1,39 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"os"
+)
+
+func printUsageAndExit() {
+	fmt.Println("Usage: ./processor <command> [--test-issue <issue name>]")
+	os.Exit(1)
+}
 
 func main() {
-	fmt.Println("Hello world")
+	if len(os.Args) < 2 {
+		printUsageAndExit()
+	}
+
+	command := os.Args[1]
+
+	if len(os.Args) == 4 {
+		if os.Args[2] == "--test-issue" {
+			testIssueName = os.Args[3]
+			testMessage(fmt.Sprintf("Using test issue '%s'", testIssueName))
+		} else {
+			printUsageAndExit()
+		}
+	} else if len(os.Args) > 4 || len(os.Args) == 3 {
+		printUsageAndExit()
+	}
```

Comment:
Nit: making the `if` statement a bit more readable. It took a bit for me to read and figure out what the control flow actually was. Feel free to disregard if future PRs will have more arg parsing and these stacked `if`s are required.

Although I do recommend using the std library [`flag`](https://pkg.go.dev/flag) package if a lot more arg parsing is required (it supports both [flags](https://gobyexample.com/command-line-flags) and [subcommands](https://gobyexample.com/command-line-subcommands)).
```suggestion
	if len(os.Args) == 4 && os.Args[2] == "--test-issue" {
		testIssueName = os.Args[3]
		testMessage(fmt.Sprintf("Using test issue '%s'", testIssueName))
	} else if len(os.Args) != 2 {
		printUsageAndExit()
	}
```